src/ui/components/shared/Onboarding/index.tsx

```diff
@@ -138,16 +137,16 @@ export function OnboardingButton({
 export function OnboardingModalContainer({
   children,
   theme = "dark",
+  visualTreatment,
 }: {
   children: React.ReactNode;
-  // For randomizing some background elements as controlled by progress
-  // on the parent component, e.g. circles/bubbles that change on click
-  randomNumber?: number;
   theme?: "dark" | "light";
+  visualTreatment?: string;
 }) {
+  const themeClass = visualTreatment ? styles[visualTreatment] : "";
```

Comment:
This seems to be unused. Remove it?
```suggestion
}) {
```

Reply:
Indeed it is, thanks!